Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
+    "\n",
+    "For most of the problems visualization of normalized velocity or pressure contours over a horizontal or vertical plane at the center of the CFD domain is a good start point for post-processing the obtained numerical data. For the current problem, since the entire domain is two dimensional and planar, the entire domain itself is the best candidate of post-processing plane. User should now define the normalized variable of interest needs to be visualized. Defining variables can be done following path of `Filters/Data Analysis/Calculator`. This command will open a new panel under `Properties`, where users can define their variable of interest; Name the variable (i.e. `U_norm`) and enter the required formula/definition for the normalized variable of interest (i.e. `U_x/1`) and hit `Apply`. This will calculate the normalized streamwise velocity with free stream velocity within the entire CFD domain for all the time steps and visualize the results for `t=0`. Using the `play` bottom users can see the formation of laminar boundary layer and wake around and in the back of the rotating cylinder:\n",
+    "\n",
+    "<img src=\"./Images/2velCountor.png\" width=\"700\" align=\"middle\">\n",
     "</br>\n",
-    "Fig. 1 - The caption for the image goes here."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Insert a YouTube video Available on the SFO Channel \n",
+    "Fig. 2 - Normalized streamwise velocity contours with the free stream velocity for flow over a rotating cylinder with \n",
+    "$Re=100$ and $TSR$=1 at `t=150`.\n",
     "\n",
-    "Create a new cell using `Edit/Split Cell` path and insert the following lines in the newly created cell and compile it, while in `Code` environment rather than `Markdown` environment.\n",
+    "Fig. 2 visualizes the normalized streamwise velocity with free stream for flows with $Re=100$ and $TSR=1$. The Flow enters the domain with normalized velocity of 1. Due to the imposed no-slip boundary conditions over the surface of cylinder, the velocity instantly decrease to zero on the nose of the cylinder and a laminar boundary layer starts to form and grow around the cylinder's surface. After few time steps due to the unsteadiness of the flow at Re=100, vortices start to form and shed from the top and bottom of the cylinder in to it's wake. This flow field behavior of the flow over a rotating cylinder with $TSR=1$ is very similar to the flow field pattern in the case of fixed cylinder in which the $TSR=0$. The only difference between these two cases is that in the case of rotating cylinder, due to the cylinder rotation, the vortex shedding frequency is lower. Increasing the TSR of the cylinder (i.e. the angular speed) the vortex shedding frequency in the wake of the cylinder gradually decreases and after a critical TSR value of the vortex shedding will completely be suppressed (i.e. vortex shedding frequency of zero). \n",
```

Comment:
Regarding this sentence "The only difference between these two cases is that in the case of rotating cylinder, due to the cylinder rotation, the vortex shedding frequency is lower." Maybe this is not the only difference since the vortex patterns seem to deviate to one side (top or down, depending on the rotation direction) and not shedding symmetrically with respect to center line.